go.mod

```diff
@@ -0,0 +1,40 @@
+module github.com/fredbi/keycloak-gatekeeper
```

Comment:
@fredbi could you please replace your username by "keycloak"? Other than that we're good to go.

Reply:
@abstractj ah yes, that's this automatic go mod update... Fixing it thanks